app/src/main/java/io/homeassistant/companion/android/onboarding/manual/ManualSetupPresenterImpl.kt

```diff
@@ -24,6 +24,10 @@ class ManualSetupPresenterImpl @Inject constructor(
             url = URL(urlString)
```

Comment:
Maybe out of scope for this PR, but it would be great if we could cut out the path of the URL, so that a user can copy paste any existing url including `/lovelace/home` in it, and we cut that part off to get the host url.

Reply:
I can do that but will open a different PR for it